protocol/bc/transaction.go

```diff
@@ -51,6 +50,7 @@ const (
 
 	// Bit mask for accepted serialization flags.
 	// All other flag bits must be 0.
+	SerTxHash   = 0x0 // this is used only for computing transaction hash - prevout and refdata are replaced with their hashes
```

Comment:
This value doesn't need to be exported. (Neither does `SerValid`, for that matter. In fact it looks like we're not using `SerValid` anywhere, which may be a bug.)